intezer_sdk/_api.py

```diff
@@ -605,6 +606,34 @@ def download_file_by_sha256(self,
                     for chunk in response.iter_content(chunk_size=8192):
                         file.write(chunk)
 
+    def download_scanner_to_stream(self, platform: str = None, path: str = None, output_stream: IO = None):
```

Comment:
You give here the option for both stream and path, so maybe have a more generic name:
`download_endpoint_scanner`?